test-app/Foo/Bar/BarAndFoo.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Test\App\Foo\Bar;
```

Comment:
Why not put this in the test namespace?

Reply:
i don't really see the difference here

Reply:
Nvm my bad, missed out we put a test app in here when switching context from another project :zipper_mouth_face: